src/content/docs/dropins/checkout/checkout-installation.mdx

```diff
@@ -1,9 +1,323 @@
+import Aside from '@components/Aside.astro';
+import Callouts from '@components/Callouts.astro';
+import Diagram from '@components/Diagram.astro';
+import { Steps } from '@astrojs/starlight/components';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';import OptionsTable from '@components/OptionsTable.astro';
+
+If you are not using the boilerplate template, you can follow the steps below to install the Checkout dropin on an existing Adobe Commerce storefront project.
+
+<Aside type="note" title="Note">
+  Before installing the Checkout dropin, make sure that you have completed the [prerequisites](/get-started/requirements/).
+</Aside>
+
+## Workflow
+
+The following diagram provides an overview of the installation process for the Checkout dropin:
+
+<Diagram>![Dropin installation](@images/pdp/pdp-installation.svg)</Diagram>
+
+## Step-by-step
+
+The following steps show how to install the Checkout dropin on your site.
+
+<Tasks>
+
+<Task>
+
+### Install the packages
+
+Use a CDN or NPM (recommended for performance) to install the dropin tools (`@dropins/tools`) and Checkout (`@dropins/storefront-checkout`) packages.
+
+<Tabs>
+  <TabItem label="NPM" icon="seti:npm">
+
+    ```bash frame="none"
+    npm install @dropins/tools @dropins/storefront-checkout
+    ```
+
+  </TabItem>
+  <TabItem label="CDN" icon="external">
+
+    ```html title="index.html" del={"Replace with actual URLs":5-7}
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Your Storefront</title>
+
+      <script src="https://cdn.jsdelivr.net/npm/@dropins/tools@latest"></script>
+      <script src="https://cdn.jsdelivr.net/npm/@dropins/storefront-checkout@latest"></script>
+    </head>
+    ```
+
+  </TabItem>
+</Tabs>
+
+:::note[Install @dropins/tools]
+All dropins require the `@dropins/tools` package. This package contains the libraries that dropins need to initialize and communicate, including the `fetch-graphql`, `event-bus`, and `initializer` utilities.
+:::
+
+</Task>
+
+<Task>
+
+### Map the packages
+
+In the `<head>` tag of your `index.html` or `head.html` file, use an `importmap` pointed to the `node_modules` directory, a **custom local directory**, or **CDN** (for run-time environments).
+
+<Tabs>
+<TabItem label="node_modules" icon="seti:npm">
+    This example shows an `importmap` added to a `head.html` The `importmap` points both packages to the local `node_modules` directory that contains your installed dropin files from the dropin tools (@dropins/tools) and the Checkout dropin (@dropins/storefront-checkout):
+  
+  ```html title="head.html"
+  <script type="importmap">
+    {
+      "imports": {
+        "@dropins/tools/": "/node_modules/@dropins/tools/",
+        "@dropins/storefront-pdp/": "/node_modules/@dropins/storefront-checkout/",
```

Comment:
I do not believe `node_modules` will work in a live site unless you host them on the web server. This is why in the boilerplate we do a post-install _copy_ to a `scripts` directory, which is then hosted by EDS:

https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/head.html#L15-L16

Do we have other examples that say to use node_modules? Those probably need fixed too.

Or I might be wrong? I just don't see how this will work in reality. I think only serving (custom local directory) or CDN will work here.

Reply:
Thanks @sirugh. The [PDP](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/product-details/pdp-installation/#map-the-packages) and [Cart](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/cart/cart-installation/#map-the-packages) installation docs say to use node_modules as well.

We're documenting three options (node_modules, custom, and CDN). Could it be that we don't need all of those options?

Reply:
It is possible I am mistaken.

But looking at the [mdn docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap#description) it seems to me like you would have to actually host that path to the directory.

As in, `"@dropins/tools/": "/node_modules/@dropins/tools/",` will look for a something at `mysite.com/node_modules/@dropins/tools` to resolve imports from @dropins/tools and unless their web server is specifically set up to host that path, it will not work.

IMO we do not need the node_modules tab. You can leave "self hosted" or CDN.

@fnhipster will you double check me :)

Reply:
For self-hosted, can we reference to something like `<path>/api.js`?